apps/node/tests/database/users/presets/group.py

```diff
@@ -0,0 +1 @@
+group_metrics = [("SysAdmins"), ("Data Analysts"), ("Hardware Owners")]
```

Comment:
This is fine for testing, but perhaps make up some university names instead to make it more accurate to reality. Groups are for grouping data scientists. You may have multiple researchers from the University of Oxford put in a group, for instance. But we won't be grouping people managing the node itself.